frontend/src/static/js/v_summary.js

```diff
@@ -389,24 +389,24 @@ window.vSummary = {
 
       this.sortFiltered();
     },
-    processFileFormats() {
+    processFileTypes() {
       const selectedColors = ['#ffe119', '#4363d8', '#3cb44b', '#f58231', '#911eb4', '#46f0f0', '#f032e6',
           '#bcf60c', '#fabebe', '#008080', '#e6beff', '#9a6324', '#fffac8', '#800000', '#aaffc3', '#808000', '#ffd8b1',
           '#000075', '#808080'];
-      const colors = {};
+      const fileTypeColors = {};
       let i = 0;
 
       this.repos.forEach((repo) => {
         repo.users.forEach((user) => {
-          Object.keys(user.fileFormatContribution).forEach((fileFormat) => {
-            if (!Object.prototype.hasOwnProperty.call(colors, fileFormat)) {
-              colors[fileFormat] = selectedColors[i];
+          Object.keys(user.fileTypeContribution).forEach((fileType) => {
+            if (!Object.prototype.hasOwnProperty.call(fileTypeColors, fileType)) {
+              fileTypeColors[fileType] = selectedColors[i];
               i = (i + 1) % selectedColors.length;
             }
           });
         });
-        this.contributionBarColors = colors;
       });
+      this.contributionBarFileTypeColors = fileTypeColors;
```

Comment:
The reason moving this out the loop?

Reply:
This one wasn't intended; probably a result of the heavy merge conflicts